src/util.ts

```diff
@@ -267,9 +267,8 @@ export function getManifestsFromRegistry(): Promise<string[]> {
       let manifestPaths = [];
 
       for (const name in registryJSON) {
-        // Manifests are inserted in the registry with matching name and value
-        if (registryJSON[name].toString().toLowerCase() === name.toString().toLowerCase()) {
-          manifestPaths.push(name);
+        if (!name.startsWith("PS")) {
```

Comment:
I don't understand this change. what is special about "PS" and why is that ignored?

Reply:
Because the command we used to use would return some Powershell property, like "PSPath".
I find another way to bypass them.